components/DeleteCandyMachine.tsx

```diff
@@ -12,17 +12,28 @@ const DeleteCandyMachine: FC<{
     const [transaction, setTransaction] = useState('')
     const { removeAccount } = useRemoveCandyMachineAccount([candyMachineAccount])
     const router = useRouter()
+    const { showNotification } = useNotification()
 
     const deleteCM = async () => {
         try {
             setIsDeleting(true)
             setStatus({ error: false, message: '' })
             const result = await removeAccount(candyMachineAccount)
             result && setTransaction(result.txid)
-            setStatus({ error: false, message: `Candy Machine deleted successfully!` })
+            showNotification({
+                open: true,
+                message: `Candy Machine deleted successfully!`,
+                type: 'success',
+                timeout: 8000,
```

Comment:
instead of adding `timeout: 8000` everywhere, cant we make the timeout with a default value so you dont have to pass it?

Reply:
I would also add a const to the lib `export const NOTIFICATION_TIMEOUT_NEVER = -1` so it can be used in case someone wants to show a notification that never autodestroys